### PR TITLE
Added block header

### PR DESCRIPTION
**Pull request by targonskii**
Added block header

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -9,7 +9,54 @@
 </head>
 <body>
     <div class="wrapper">
```
Comment (ghost) [inline Triphouse-project/index.html]: 2021-04-15_23h56_06

Header should has properties.

Comment (ghost) [reply Triphouse-project/index.html]: and margin: 0px auto

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -9,7 +9,54 @@
...
-        <header></header>
+        <header>
```
Comment (ghost) [inline Triphouse-project/index.html:11]: 2021-04-15_23h56_32
should  has properties.

File: Triphouse-project/index.html (hunk 1 above)
Comment (targonskii) [reply Triphouse-project/index.html]: Fixed it

File: Triphouse-project/index.html (hunk 2 above)
Comment (targonskii) [reply Triphouse-project/index.html:11]: Changed properties.

**Pull request merged**